jest-setup.js

```diff
@@ -2,4 +2,10 @@ import { NativeModules } from 'react-native';
 
 NativeModules.ReactLocalization = {
   language: 'en',
-};
\ No newline at end of file
+};
+
+jest.mock('react-native-device-info', () => {
+  return {
+    getUniqueId: jest.fn(),
+  };
+});
```

Comment:
I think it looks more readable
```suggestion
jest.mock('react-native-device-info', () => ({
    getUniqueId: jest.fn(),
  }));
```